addons/account/wizard/account_move_reversal.py

```diff
@@ -12,6 +13,7 @@ class AccountMoveReversal(models.TransientModel):
 
     move_id = fields.Many2one('account.move', string='Journal Entry',
```

Comment:
move_id can be removed, isn't it?

Reply:
If we are sure it won't be used elsewhere than in `helpesk_account`, I can move it there and put 
```python3
@api.onchange('move_id')
def _onchange_move_id(self):
    self.move_ids = self.move_id
```
but we still need a m2o field on the form view.